deps.qt/qt6.zsh

```diff
@@ -18,6 +18,7 @@ local -a qt_components=(
   'qtshadertools'
   'qtmultimedia'
   'qtsvg'
+  'qtcharts'
```

Comment:
Does qtcharts require anything other than qtbase? If not, then it should be placed just below qtbase.

Reply:
fixed